### PR TITLE
C-API Server start definitions

### PR DESCRIPTION
**Pull request by dkalinowski**

### CODE REVIEW

File: src/pocapi_serverstart.hpp [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+void OVMS_ServerMultiModelOptionsDelete(OVMS_ServerMultiModelOptions* options);
+
+// --config_path
+void OVMS_ServerMultiModelOptionsSetConfigPath(OVMS_ServerMultiModelOptions* options,
+    const char* config_path);
+
+
+////
+//// OVMS_Server
+//// Handler for all managerment activities
```
Comment (atobiszei) [inline src/pocapi_serverstart.hpp:87]:
```suggestion
//// Handler for all management activities
```

File: src/pocapi_serverstart.hpp [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+//// OVMS_ServerGeneralOptions
+//// Structure for general options for both: single and multi (with config.json) management
+////
+// Allocates memory for server general options and returns ptr
+void OVMS_ServerGeneralOptionsNew(OVMS_ServerGeneralOptions** options);
+// Deallocates server general options memory for given ptr
+void OVMS_ServerGeneralOptionsDelete(OVMS_ServerGeneralOptions* options);
+
+// --log_level
+void OVMS_ServerGeneralOptionsSetLogLevel(OVMS_ServerGeneralOptions* options,
```
Comment (atobiszei) [inline src/pocapi_serverstart.hpp:43]: return OVMS_Status in all functions

**Comment by atobiszei** [comment]
Move those declarations into pocapi.hpp for now.

File: src/pocapi_serverstart.hpp (hunk 1 above)
Comment (dkalinowski) [reply src/pocapi_serverstart.hpp]: done

**Comment by dkalinowski** [comment]
> Move those declarations into pocapi.hpp for now.

Done. This will create conflict with your PR introducing calls for inference. Whoever merges seconds needs to take care of that.

**Review by atobiszei — APPROVED** [review]

**Pull request merged**